content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md

```diff
@@ -22,7 +22,10 @@ For more detailed information, see [Receiving update notifications](https://docu
 
 The [Constant Value](https://cumulocity.com/guides/10.19.0/streaming-analytics/block-reference/#constant-value) block now supports `float` and `boolean` value types
 and can produce output of these types. This enables the block's output to be consumed by other blocks that take input of type `float` or `boolean` like the blocks in
-the **Logic** and **Aggregate** categories. The **Type** parameter is also now optional. If a type is not selected, the type of the output value is inferred from the **Value** parameter. 
+the **Logic** and **Aggregate** categories. The **Type** parameter is also now optional. If a type is not selected, the type of the output value is inferred from the **Value** parameter.
+
+An icon is now provided for removing a template parameter from the **Template Parameter** dialog box.
```

Comment:
We do no longer talk about guides in the context of the new documentation website. You could say "in the Streaming Analytics section" or simply "in the user documentation"

Reply:
Thanks Beate. I've change this to "user documentation". Note that I did not touch the other places where you already changed the links to the other guides on your other PR. Maybe you also want to do this on #303.